resources/views/api/includes/sections/top_nav_user.blade.php

```diff
@@ -9,20 +10,6 @@
             </a>
             <ul class="right hide-on-med-and-down">
                 <li>
-                    <form action="{{ route('searchsuya') }}" role="search">
-
-                        <div class="input-field">
-                            <input name = "query" id="query" type="search" class="search" required>
-                            <label for="search">
-                                <i class="material-icons teal-text text-lighten-2">search</i>
-                            </label>
-                            <button type="submit" style="display:none;" name="search">search</button>
-                            <i class="material-icons">close</i>
-                        </div>
-                    </form>
-                </li>
-
-                <li>
                     @if (  Auth::check() )
```

Comment:
**Nitpick**

Please remove the extra spaces around Auth::check()..It should rather be `@if (Auth::check())`